tests/pre_processing/test_extract_landscape.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import unittest
+import os.path
+
+from src.pre_processing.extract_landscape import get_facial_vectors, _extract_photos_from_file
```

Comment:
soooooo
1) separated imports *generally* make for less git conflicts in high churn files - this is preference/code style, **not** a requirement
2) i'll let the _method slide because this is a test class, which, btw, AWESOME :+1: